spec/lib/polariscope/scanner/gemfile_health_score_spec.rb

```diff
@@ -20,23 +20,43 @@
     it { is_expected.to be_nil }
   end
 
-  context 'when dependencies found and some new versions ignored' do
-    let(:gemfile_path) { File.join(Dir.pwd, 'spec/files/gemfile_with_dependencies') }
-    let(:gemfile_lock_content) { File.read('spec/files/gemfile.lock_with_dependencies') }
-    let(:bundler_audit_config_path) { File.join(Dir.pwd, 'spec/files/bundler-audit') }
+  # these tests run in the order they're defined and assert that each next test
+  # has a score lower than the previous one
+  describe 'scores', order: :defined do
+    last_score = nil
 
-    it 'returns a score' do
-      expect(score).to be <= 43.32
+    context 'when dependencies found and some new versions ignored' do
+      let(:gemfile_path) { File.join(Dir.pwd, 'spec/files/gemfile_with_dependencies') }
+      let(:gemfile_lock_content) { File.read('spec/files/gemfile.lock_with_dependencies') }
+      let(:bundler_audit_config_path) { File.join(Dir.pwd, 'spec/files/bundler-audit') }
+
+      it 'returns a score' do
+        expect(score).to be <= 43.32
```

Comment:
```suggestion
        expect(score).to be <= last_score
```

should we do this then?

Reply:
We don't have a `last_score` at this point since it's the first test. For the first one we have to assert for a concrete value, and for later tests we only need to assert that the score is less than the previous one.